Tests/SonarRunner.Shim.Tests/SonarRunnerWrapperTests.cs

```diff
@@ -28,37 +27,37 @@ public void SonarRunnerHome_NoWarning()
             string path = Path.Combine(TestUtils.CreateTestSpecificFolder(TestContext), "analysis.properties");
             File.CreateText(path);
 
-            // Act
-            SonarRunnerWrapper.ExecuteJavaRunner(testLogger, exePath, exePath);
+            using (EnvironmentVariableScope scope = new EnvironmentVariableScope())
```

Comment:
The EnvironmentVariableScope is the standard approach used in tests to isolate the managed tests from the environment of the host machine. It tracks the environment variables that are set in the test and automatically resets them to their original values when the scope is disposed.